### PR TITLE
Update GUIDELINES.md

### PR DESCRIPTION
**Pull request by johanneskoester**
* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [x] This PR does something else (explain below).

### CODE REVIEW

File: GUIDELINES.md [hunk 1]
```diff
@@ -72,13 +72,12 @@ recipes. Recipes for dependencies with an `r-` prefix should be created using
 
 Add a wrapper script if the software is typically called via `java -jar ...`.
 For example, [fastqc](recipes/fastqc) already has a wrapper script, but
-[gatk-framework](recipes/gatk-framework) does not. [chromhmm](recipes/chromhmm)
-mimics the installation pattern of gatk-framework.
+[peptide-shaker](recipes/peptide-shaker) does not.
```
Comment (bgruening) [inline GUIDELINES.md]: Peptide Shaker has a wrapper script: https://github.com/bioconda/bioconda-recipes/tree/master/recipes/peptide-shaker
The new python one.

File: GUIDELINES.md [hunk 2]
```diff
@@ -72,13 +72,12 @@ recipes. Recipes for dependencies with an `r-` prefix should be created using
...
 
 JAR files should go in `$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM`.
 A wrapper script should be placed here as well, and symlinked to `$PREFIX/bin`.
 
-Example: [gatk-framework](recipes/gatk-framework)
```
Comment (bgruening) [inline GUIDELINES.md]: I guess this should be a list, otherwise github renders it as one line.

File: GUIDELINES.md (hunk 1 above)
Comment (johanneskoester) [inline GUIDELINES.md]: Yes, what this means is that peptide-shaker has a wrapper that we had to provide. Fastqc provides its own wrapper.

File: GUIDELINES.md (hunk 2 above)
Comment (johanneskoester) [inline GUIDELINES.md]: Indeed, thanks!

**Pull request merged**